src/components/PostPageDetail/index.js

```diff
@@ -40,6 +40,7 @@ import WriteComment from '../Comments/WriteComment';
 import useWriteComment from '../Comments/hooks/useWriteComment';
 import LoadingWithoutModal from '../LoadingWithoutModal';
 import {withInteractionsManaged} from '../WithInteractionManaged';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the `theme` module is a good change. It centralizes the color definitions, making it easier to manage and update colors across the application.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->